with-nextjs/.env.example

```diff
@@ -0,0 +1,16 @@
+# https://docs.polar.sh/integrate/oat
+NEXT_PUBLIC_PRODUCTION_POLAR_ACCESS_TOKEN=
+
+# https://docs.polar.sh/integrate/webhooks/endpoints#setup-webhooks
+NEXT_PUBLIC_PRODUCTION_POLAR_WEBHOOK_SECRET=
+
+# use the above same approach to get the sandbox credentials.
+NEXT_PUBLIC_SANDBOX_POLAR_ACCESS_TOKEN=
+
+NEXT_PUBLIC_SANDBOX_POLAR_WEBHOOK_SECRET=
```

Comment:
this can not be public, since the token is supposed to be on the server side only